home-manager/pgp.nix

```diff
@@ -1,6 +1,25 @@
-{ config, pkgs, ... }:
+{
+  config,
+  pkgs,
+  edge-pkgs,
+  ...
+}:
 
-# ## FAQ - GPG
+# # FAQ
+#
+# ## sequoia-sq and gpg
+#
+# See GH-830
+#
+# If you faced to decrypt error with gpg-sq, check it with `sq decrypt`. It displays error details.
+# For example, `1: AEAD Encrypted Data Packet v1 is not considered secure`
+# This is caused by encrypted non configured gpg for the AEAD. Disable it with showpref/setpref if you still use gpg.
```

Comment:
This operation is cannot be done with gpg-sq, because of --edit-key is not yet implemented https://gitlab.com/sequoia-pgp/sequoia-chameleon-gnupg/-/issues/92